plugins/navigation-files-transformer/gatsby-node.js

```diff
@@ -49,11 +45,25 @@ const onCreateNode = async (api, options) => {
       children: [],
       parent: node.id,
       internal: {
+        content: "",
         contentDigest: createContentDigest(parsedContent),
-        type: "parliamentNavigation",
+        type: "ParliamentNavigation",
```

Comment:
what's the reason for the capitalization? doesn't this go against the naming convention for Gatsby's GraphQL types?

Reply:
If I set it to lower case and then I query all the types I get:

```
File
Directory
SitePage
Site
ImageSharp
Mdx
parliamentNavigation
DocsYaml
SiteBuildMetadata
SitePlugin
MarkdownRemark
```

So it looks out of place which is why I went with `ParliamentNavigation`.

Reply:
ah, ok 👍